src/ontology/mondo-ingest.Makefile

```diff
@@ -351,23 +354,43 @@ deploy-mondo-ingest:
 
 USE_MONDO_RELEASE=false
 
+# TODO: explain what does here
```

Comment:
## Comments as documentation
- [ ] @joeflack4 todo

I undrafted the PR because this is quick and otherwise it is ready. I'll just add some comments about what `tmp/mondo_repo_built` and `refresh-mondo-clone`, just because they look really confusing and are otherwise idiosyncratic.

Reply:
done